src/lunabot_localisation/test/test_localisation_launch.py

```
@@ -109,10 +109,14 @@ def test_validate_boolean_launch_arguments_rejects_invalid_launch_value():
 
-def test_generate_launch_description_has_validation_and_one_tag_pose_node():
+def test_generate_launch_description_has_validation_and_one_tag_pose_node(
+    monkeypatch: pytest.MonkeyPatch,
+):
     launch_module = _load_launch_module()
-    launch_module.get_package_share_directory = (
-        lambda _package: "/tmp/lunabot_localisation"
+    monkeypatch.setattr(
+        launch_module,
+        "get_package_share_directory",
+        lambda _package: "/tmp/lunabot_localisation",
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/lunabot_localisation"

(S108)